crates/next-core/src/base_loader_tree.rs

```diff
@@ -99,7 +99,7 @@ impl BaseLoaderTreeBuilder {
         self.imports.push(
             formatdoc!(
                 r#"
-                import * as {} from "MODULE_{}";
+                const {} = () => import("MODULE_{}");
```

Comment:
That's not exactly matching webpack behavior. Webpack uses `webpackMode: "eager"` which avoid creating dynamic chunks for this. But we don't support this mode for Turbopack yet.